src/model/databaseNode.ts

```diff
@@ -36,6 +38,11 @@ export class DatabaseNode implements INode {
 
         return Utility.queryPromise<any[]>(connection, `SELECT TABLE_NAME FROM information_schema.TABLES  WHERE TABLE_SCHEMA = '${this.database}' LIMIT ${Utility.maxTableCount}`)

```

Comment:
If this.keyword is set, how about updating the SQL to add `TABLE_NAME like '%this.keyword%'`? In case there are over thousands of tables in a database.